### ISSUE TITLE
Macos tel link doesn't dial or use number

### ISSUE THREAD

**Issue by ibroom** [BelledonneCommunications/linphone-desktop#517]
Hi everyone

Amazing work on a great SIP client for Mac! I have recently recommended it instead of our previous Mac app 'telephone' and colleagues love how clear the audio is.

We have been trying to replace the existing app by associating 'tel' links with Linphone via Facetime. This means Linphone opens when tel links are clicked in a browser but the phone number in the URL is not populated nor does Linphone dial the number provided. Is this intentional?

For example, entering the following URL into Chrome loads Linphone but doesn't dial but I would have thought it should. Other apps pre-load the number ready for the user to click 'dial':
tel:+123456789

Thanks for your assistance.

**Comment by awnz** [comment]
Same on Linux (me: Ubuntu 20.04). The Linphone console log shows this when I click a tel: link in the browser (phone number masked out)...

```
[09:31:26:480][0x2797770][Warning]"This command doesn't exist: `tel`."
[09:31:26:480][0x2797770][Info]"Detecting URI command: `tel:*********`..."
[09:31:26:480][0x2797770][Info]"Execute command:" QHash(("sip-address", "sip:*********"))
[09:33:01:124][0x2797770][Info]"Received command from other application: `show`."
[09:33:01:124][0x2797770][Info]"Detecting cli command: `show`..."
[09:33:01:124][0x2797770][Info]"Execute command:" QHash()
```

... but nothing happens, no call is placed, nothing happens in the UI at all. If in the background, it doesn't even come to the foreground despite the logging above around "show".

If I manually execute the Linphone AppImage with the parameter "call sip-address=*********" (including quotes) it works as intended, but this isn't achievable with the usual protocol handler registering which passes a tel: link as above.

**Comment by awnz** [comment]
Looks like this is a duplicate of #513. Moving my comment there.

**Comment by julonexus** [comment]
Hi
The things is that passing from URL, it will try to call sip:123 that is not a sip address. So the call is not done.
When calling from CLI, it will try to call 123, that can be transform into a sip address. (sip:123@domain)
A fix is available on 5.0.
